### PR TITLE
MODINVSTOR-573: Return 422 when service point user does not exist.

### PR DESCRIPTION
**Pull request by bohdan-suprun**
https://issues.folio.org/browse/MODINVSTOR-573 - Unable to upgrade inventory-storage on rancher env

### CODE REVIEW

File: ramls/service-points-user.raml [hunk 1]
```diff
@@ -10,7 +10,7 @@ documentation:
 
 types:
   servicePointsUser: !include servicepointsuser.json
-  servicepointsusers: !include servicepointsusers.json
+  servicePointsUsers: !include servicepointsusers.json
```
Comment (marcjohnson-kint) [inline ramls/service-points-user.raml:13]: Why has the case of this name been changed?

File: src/test/java/org/folio/rest/api/ServicePointsUserTest.java [hunk 2]
```diff
@@ -164,20 +157,36 @@ public void canUpdateServicePointUser() throws MalformedURLException,
...
     createServicePointUser(spuId, UUID.randomUUID(), null, null);
     Response getResponse = getServicePointUserById(spuId);
     assertThat(getResponse.getStatusCode(), is(200));
     deleteServicePointUserById(spuId);
     Response getResponseAgain = getServicePointUserById(spuId);
     assertThat(getResponseAgain.getStatusCode(), is(404));
+  }
+
+  @Test
+  public void cannotCreateServicePointUserIfUserAlreadyExist() throws Exception {
```
Comment (marcjohnson-kint) [inline src/test/java/org/folio/rest/api/ServicePointsUserTest.java:176]:
```suggestion
  public void cannotCreateServicePointUserIfUserAlreadyExists() throws Exception {
```

**Review by marcjohnson-kint — APPROVED** [review]

File: ramls/service-points-user.raml (hunk 1 above)
Comment (bohdan-suprun) [reply ramls/service-points-user.raml:13]: In order to have the class generated with correct camel style.

Comment (marcjohnson-kint) [reply ramls/service-points-user.raml:13]: Ah, I didn't know that.

**Pull request merged**